mmdet/models/data_preprocessors/data_preprocessor.py

```diff
@@ -368,6 +368,87 @@ def forward(
 class MultiBranchDataPreprocessor(BaseDataPreprocessor):
     """DataPreprocessor wrapper for multi-branch data.
 
+    Take semi-supervised object detection as an example, assume that
+    the ratio of labeled data and unlabeled data in a batch is 1:2,
+    `sup` indicates the branch where the labeled data is augmented,
+    `unsup_teacher` and `unsup_student` indicate the branches where
+    the unlabeled data is augmented by different pipeline. Therefore,
+    the input format of multi-branch data is shown as follows:
+
```

Comment:
It is not very clear which of the following examples are inputs, outputs, and intermediate results of which step.